h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -1747,6 +1747,11 @@ public Frame sort( int[] cols ) {
   public Frame sort(int[] cols, int[] ascending) {
     return Merge.sort(this, cols, ascending);
   }
+  
+  public void label(final String labelColumnName){
```

Comment:
Such a method imho doesn't belong in the Frame class.

Reply:
Moved to FrameUtils, thanks.